payment/controllers.go

```diff
@@ -445,6 +462,36 @@ func GetOrderCreds(service *Service) handlers.AppHandler {
 			)
 		}
 
+		order, err := service.Datastore.GetOrder(*orderID.UUID())
+
+		var credentials []TimeLimitedCreds
+		timeLimitedSecret := cryptography.NewTimeLimitedSecret([]byte(os.Getenv("BRAVE_MERCHANT_KEY")))
```

Comment:
there should be a guard to make sure we're only generating time-limited credentials for merchant = brave